src/main/kotlin/sg/com/quantai/etl/services/TimescaleDBTableInitializer.kt

```diff
@@ -34,37 +34,45 @@ class TimescaleDBTableInitializer(val jdbcTemplate: JdbcTemplate) {
     fun runDatabaseInitializer(): CommandLineRunner {
         return CommandLineRunner {
 
-            // Start of table initialization
             logger.info("Initializing all required tables for TimescaleDB...")
 
-            // Initialize `raw_crypto_data` table
             logger.info("Initializing 'raw_crypto_data' table...")
```

Comment:
To follow the pattern created by Karthik during the newsArticle, we should:
- Instead of calling the table "raw_XXX", we can name it after its source. Therefore, we can rename this table as "cryptoCryptoCompareAPI" or "crypto_CryptoCompareAPI"